crates/hyperqueue/src/common/serverdir.rs

```diff
@@ -207,7 +207,8 @@ pub struct ConnectAccessRecordPart {
 
 impl FullAccessRecord {
     pub fn new(
-        host: String,
+        client_host: String,
```

Comment:
nit: this function could now receive `ConnectAccessRecordPart` directly, to avoid sending the fields one by one.